examples/custom-ini.jsonnet

```diff
@@ -15,14 +15,15 @@ local grafana = ((import 'grafana/grafana.libsonnet') + {
                    },
                  }).grafana;
 
-k.core.v1.list.new([
-  grafana.config,
-  grafana.dashboardDefinitions,
-  grafana.dashboardSources,
-  grafana.dashboardDatasources,
-  grafana.deployment,
-  grafana.serviceAccount,
-  grafana.service +
-  service.mixin.spec.withPorts(servicePort.newNamed('http', 3000, 'http') + servicePort.withNodePort(30910)) +
-  service.mixin.spec.withType('NodePort'),
-])
+k.core.v1.list.new(
+  [dashboard for dashboard in grafana.dashboardDefinitions] + [
```

Comment:
`grafana.dashboardDefinitions` is already an array, it should be possible to just concatenate the two arrays without the `for`.